mlrun/db/httpdb.py

```diff
@@ -580,6 +580,7 @@ def connect(self, secrets=None):
                 or config.feature_store.default_targets
             )
             config.alerts.mode = server_cfg.get("alerts_mode") or config.alerts.mode
+            config.system_id = server_cfg.get("system_id")
```

Comment:
Why not?
```suggestion
            config.system_id = server_cfg.get("system_id") or config.system_id
```

Reply:
@TomerShor Won't we always have it always in the server_cfg ?